docs/develop/make_it_your_own.md

```diff
@@ -0,0 +1,443 @@
+<!-- // Paste your code here -->
+{%- endblock page_body %}
+```
+
+Inside the `page_body` block you can restructure the page as you want! You can check the default record landing page template [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/theme/templates/invenio_rdm_records/record_landing_page.html).
+
+Since we modified `invenio.cfg`, we need to re-start the server to see our changes take effect.
+
+
+## Define a custom controlled vocabulary
+
+InvenioRDM deals with many controlled vocabularies. And since agreeing on standard
+vocabularies is notoriously difficult, InvenioRDM allows you to use the controlled
+vocabularies **you** want. Here we show how to customize your resource types vocabulary,
+but the same approach holds true for any other vocabulary.
+
+We start by copying [the default resource types vocabulary file](https://github.com/inveniosoftware/invenio-rdm-records/raw/master/invenio_rdm_records/vocabularies/resource_types.csv)
+and modifying it with our own vocabulary entries. Make sure to keep the same headers!
+
+!!! info "Use the hierarchy to your advantage"
+    You will notice the hierarchy in the default file. It reduces repetition and makes sections visually clear. You can use that pattern for your own entries too.
+
+Save this file in the `app_data/vocabularies/` folder. We call ours `resource_types.csv`. Very original!
+Then edit `invenio.cfg` to tell InvenioRDM to use this file.
+
+```python
+# imports at the top...
+from os.path import abspath, dirname, join
+
+# ...
+
+# At the bottom
+# Our custom Vocabularies
+RDM_RECORDS_CUSTOM_VOCABULARIES = {
+    'resource_types': join(
+        dirname(abspath(__file__)),
+        'app_data', 'vocabularies', 'resource_types.csv')
+}
+```
+
+!!! info "Other vocabularies"
+    See the [Vocabulary source code](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/vocabularies/__init__.py)
+    in invenio-rdm-records to know the keys for the other vocabularies.
+
+Restart your server and your vocabulary will now be used for resource types!
+
+
+## Extend the metadata model
+
+We've designed the default InvenioRDM metadata model to incorporate much of the
+useful fields the digital repository community has adopted over the years. From
+Subjects to Language to Location fields, there is a lot of depth to the out-of-the-box
+model. We encourage you to explore it fully before you consider adding your own
+fields. But adding your own fields is possible and no hacks are necessary.
+
+Metadata extensions are defined via two configurations: the additional fields'
+namespaces, a unique identifying url to prevent field clashes with other extensions,
+and the extensions (fields) themselves, a name, validation schema type and
+ElasticSearch storage type.
+
+To extend the metadata model, we edit the corresponding configuration variables
+in our familiar friend `invenio.cfg`. For our purposes, we first add the new
```

Comment:
minor: I understand "For our purposes" is because it could be using an already existing namespace? This is not 100% clear, maybe a small point to why? or to "you can use existing namespaces, see which ones exist [here]().

Reply:
I meant it as "For example":

```suggestion
in our familiar friend `invenio.cfg`. For example, we first add the new
```